README.md

```diff
@@ -24,6 +24,46 @@ poetry install
 ```
 You can then get a shell with the environment via `poetry shell`.
 
+---
+**NOTE**
+
+Following the steps above will install the package with only Numpy support. To use it with, for example, GPflow (TensorFlow) or GPyTorch (PyTorch), follow the additional steps below.
+
+---
+
+## Next, choose the backend
+
+Geometric kernels can use `tensorflow`, `pytorch`, ~~`jax`~~ (under development) and `numpy` (default) as backend thanks to [EagerPy](https://github.com/jonasrauber/eagerpy). To install the required dependencies for this run the following command
+
+- For TensorFlow with GPflow
+```
+poetry install -E tensorflow
+```
+and export
+```
+export GEOMETRIC_KERNELS_BACKEND=tensorflow
+```
+
+- Or, for PyTorch with GPyTorch
+```
+poetry install -E pytorch
```

Comment:
When running this, I am getting the following error:

`Installing black (20.8b1): Failed

  EnvCommandError`

I have python 3.8.10, and googling the error above seems to be related to that. Not sure I should report this as an issue, but I still wanted to have it here.